### DISCUSSION TITLE
Which CSS Minifier tool does Bootstrap 5 use?

### DISCUSSION THREAD

**Discussion by coudrak** [twbs/bootstrap#38698] [General] [answered]
Hi!

When I try to minify the CSS of Bootstrap 5, most of tools like https://www.cleancss.com/css-minify/ fail with errors (probably because of variables).

I found this one which works to generate a result https://www.toptal.com/developers/cssminifier BUT it changes the order of CSS, and for example the classes **col** or **col-12** have priority on the classes **col-md-...** on desktop! It breaks the grid on desktop...

The minified CSS file provided with Bootstrap  in /dist/ works, so I wonder which tool is used to generate it?

Thanks!

**Comment by julien-deramond** [comment, marked as answer]
Hi @coudrak 
We use [clean-css](https://github.com/clean-css/clean-css) via [clean-css-cli](https://github.com/clean-css/clean-css-cli) in our _package.json_:

https://github.com/twbs/bootstrap/blob/7a4226875d23f78ca52b2c604a9d2073f3abbba4/package.json#L51

**Reply by coudrak** [reply]
Ah ok, thanks!
